internal/cli/atlas/alerts/acknowledge.go

```diff
@@ -23,7 +23,7 @@ import (
 	"github.com/mongodb/mongodb-atlas-cli/internal/cli/require"
 	"github.com/mongodb/mongodb-atlas-cli/internal/config"
 	"github.com/mongodb/mongodb-atlas-cli/internal/flag"
-	"github.com/mongodb/mongodb-atlas-cli/internal/store"
+	atlasStore "github.com/mongodb/mongodb-atlas-cli/internal/store/atlas"
```

Comment:
any reason to include atlas in the name? keeping it `store` reduces the number of changes and I don't belive there be a case were you will include both stores right?
```suggestion
	store "github.com/mongodb/mongodb-atlas-cli/internal/store/atlas"
```

Reply:
Yes. No 2 stores should be present.
I wanted to make sure we aren't confusing stores (duplicated files). 
I wanted to make sure it is easy to search replace. 
store as identifier is used everywhere like `ops.store` etc. 
store as name is failry generic so it would be actually much harder to replace it.
Will do change but it feels like more explicit is better.